app/manager-frontend/src/pages/ActivitySource/index.jsx

```diff
@@ -0,0 +1,121 @@
+import {
+  VStack,
+  Input,
+  FormLabel,
+  Box,
+  Select,
+  Button,
+  Heading,
+  Container,
+  Alert,
+  AlertIcon,
+} from "@chakra-ui/react";
+import React, { useState } from "react";
+import { Form, Formik, useField } from "formik";
+import { FaArrowRight } from "react-icons/fa";
+import { useBackendApi } from "contexts/BackendApi";
+import { FormikInput } from "../../components/forms/FormikInput";
+import { FormikSelect } from "../../components/forms/FormikSelect";
+import { useNavigate } from "react-router-dom";
+import { validationSchema } from "./validation";
+
+export const ActivitySource = () => {
+  // I assume I should be getting this from the backend
```

Comment:
-
```suggestion
  // TODO: Get this from the backend
```

📌👍yes, good shout, can we create a PBI for it; it'll need an endpoint as well as amending this file